src/app/modules/item/pages/item-display/item-display.component.ts

```diff
@@ -117,7 +124,7 @@ export class ItemDisplayComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   updateHeight(): void {
```

Comment:
For later probably:
Hard to get when seeing `updateHeight, if this function change the height of task, change immediately the height of the display or schedule the update of height of the display in accordance with the height of the task

Reply:
I'd rename it to `updateIframeHeight` if that sounds clearer.

Reply:
I tried to guess what  `updateIframeHeight` would mean before checking at the code... and it was not really doing what I expected. The minimum would be to add explicit comments (even short). I think I would prefer a more explicit name, as `updateIframeToTaskHeight`.

Reply:
This will be addressed in the next PR (which will probably remove the function altogether).